### PR TITLE
New version: wethat.onenotetaggingkit version 4.0.30319

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/57276)

### CODE REVIEW

File: manifests/w/wethat/onenotetaggingkit/4.0.30319/wethat.onenotetaggingkit.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: wethat.onenotetaggingkit
+PackageVersion: 4.0.30319
```
Comment (ItzLevvie) [inline manifests/w/wethat/onenotetaggingkit/4.0.30319/wethat.onenotetaggingkit.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.0.8134.39795
```

File: manifests/w/wethat/onenotetaggingkit/4.0.30319/wethat.onenotetaggingkit.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: wethat.onenotetaggingkit
+PackageVersion: 4.0.30319
```
Comment (ItzLevvie) [inline manifests/w/wethat/onenotetaggingkit/4.0.30319/wethat.onenotetaggingkit.installer.yaml:5]:
```suggestion
PackageVersion: 4.0.8134.39795
```

File: manifests/w/wethat/onenotetaggingkit/4.0.30319/wethat.onenotetaggingkit.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: wethat.onenotetaggingkit
+PackageVersion: 4.0.30319
```
Comment (ItzLevvie) [inline manifests/w/wethat/onenotetaggingkit/4.0.30319/wethat.onenotetaggingkit.yaml:5]:
```suggestion
PackageVersion: 4.0.8134.39795
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/w/wethat/onenotetaggingkit/4.0.30319/wethat.onenotetaggingkit.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/w/wethat/onenotetaggingkit/4.0.30319/wethat.onenotetaggingkit.locale.en-US.yaml:5]: to match ARP.

File: manifests/w/wethat/onenotetaggingkit/4.0.30319/wethat.onenotetaggingkit.installer.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/w/wethat/onenotetaggingkit/4.0.30319/wethat.onenotetaggingkit.installer.yaml:5]: to match ARP.

File: manifests/w/wethat/onenotetaggingkit/4.0.30319/wethat.onenotetaggingkit.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/w/wethat/onenotetaggingkit/4.0.30319/wethat.onenotetaggingkit.yaml:5]: to match ARP.

**Review by OfficialEsco — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**